arviz/stats/stats.py

```diff
@@ -1322,12 +1329,12 @@ def summary(
         dfs = []
         for var_name, values in joined.data_vars.items():
             if len(values.shape[1:]):
-                metric = list(values.metric.values)
```

Comment:
This `metric` assignment clashed with the `metric` assignment on line 1307 in the original file: `for metric, circ_stat in zip(...)`. This was a type error since the two assignments to `metric` had different types.

Renaming this `metric` to `index_metric` resolves the problem since now the differently-typed values are assigned to different variables.